tests/integration/suite_test.go

```diff
@@ -112,20 +116,21 @@ var _ = AfterSuite(func() {
 })
 
 var _ = BeforeEach(func() {
-	testNamespace = createTestNamespace(context.Background())
+	schemaVersion = "1.0.1"
```

Comment:
This is a bit confusing because the schema version for installations, credsets and param sets can all be different (though I think the stars aligned recently and they are all the same for now).